src/common/telemetry_ops/app_telemetry.rs

```diff
@@ -168,6 +192,7 @@ impl Anonymize for RunningEnvironmentTelemetry {
             ram_size: self.ram_size.anonymize(),
             disk_size: self.disk_size.anonymize(),
             cpu_flags: self.cpu_flags.clone(),
+            cpu_endian: None,
```

Comment:
Why do we clear endian here? Is an endian a sensitive information? I propose to provide an info about endian from users because we send CPU flags, linux distribution, number of cores - which are maybe more sensitive

Reply:
For the sake of consistency with the other fields we have there - I'll not anonymize it.